linux/debian/8/foxpass_setup.py

```diff
@@ -39,19 +39,31 @@ def main():
     parser.add_argument('--bind-user', required=True, help='Bind User')
     parser.add_argument('--bind-pw', required=True, help='Bind Password')
     parser.add_argument('--api-key', required=True, help='API Key')
-    parser.add_argument('--ldap-uri', default='ldaps://ldap.foxpass.com', help='LDAP Server')
-    parser.add_argument('--api-url', default='https://api.foxpass.com', help='API Url')
+    parser.add_argument('--ldap-uri', '--ldap', default='ldaps://ldap.foxpass.com', help='LDAP Server')
+    parser.add_argument('--secondary-ldap', dest='ldaps', default=[], action='append', help='Secondary LDAP Server(s)')
+    parser.add_argument('--api-url', '--api', default='https://api.foxpass.com', help='API Url')
+    parser.add_argument('--secondary-api', dest='apis', default=[], action='append', help='Secondary API Server(s)')
     parser.add_argument('--ldap-connections', default=2, help='Number of connections to make to LDAP server.')
 
     args = parser.parse_args()
 
     binddn = 'cn=%s,%s' % (args.bind_user, args.base_dn)
 
+    uris = ['uri %s' % args.ldap_uri]
+    for uri in args.ldaps:
+        uris.append('uri %s' % uri)
+
+    curls = ['curl -s -q -m 5 -f "%s/sshkeys/?secret=${secret}&user=${user}&hostname=${hostname}&aws_instance_id=${aws_instance_id}" 2>/dev/null' % args.api_url]
+    for api in args.apis:
+        apis.append('curl -s -q -m 5 -f "%s/sshkeys/?secret=${secret}&user=${user}&hostname=${hostname}&aws_instance_id=${aws_instance_id}" 2>/dev/null' % api)
+
+    seperator = ' || '
+
     apt_get_update()
     install_dependencies()
-    write_foxpass_ssh_keys_script(args.api_url, args.api_key)
-    write_nslcd_conf(uri=args.ldap_uri, basedn=args.base_dn, binddn=binddn, bindpw=args.bind_pw,
-                     threads=int(args.ldap_connections))
+    write_foxpass_ssh_keys_script(curls, args.api_key, seperator)
+    write_nslcd_conf(uris=uris, secondary_ldap=secondary_ldap, basedn=args.base_dn,
```

Comment:
secondary_ldap is not used. move separator variable into write_foxpass_ssh_keys_script function (and fix spelling).